src/expectEvent.js

```diff
@@ -115,19 +118,19 @@ function isBN (object) {
 }
 
 function isWeb3Receipt (receipt) {
-  return 'events' in receipt;
+  return 'events' in receipt && typeof receipt.events === 'object';
```

Comment:
It's possible to simplify this down to only the second part of the test, because if `receipt` doesn't have an `events` key, `typeof receipt.events` will be `'undefined'`.

Reply:
I considered that, but thought that by checking for the type, the additional meaning of 'does this property exist' was not immediately obvious.